Libs/MRML/Core/vtkMRMLNode.h

```diff
@@ -333,13 +333,33 @@ class VTK_MRML_EXPORT vtkMRMLNode : public vtkObject
   ///
   /// If set to NULL, multiple instances of this node class are allowed.
   ///
-  /// If set to a non-NULL string, the node will be a singletong and
+  /// If set to a non-NULL string, the node will be a singleton and
   /// the scene will replace this node instead of adding new instances.
   ///
   /// The SingletonTag is used by the scene to build a unique ID.
+  ///
+  /// If the there can only be one instance of a given node class in the scene,
+  /// then the singleton tag should be Singleton.
+  /// If the singleton node is associated with a specific module it should be
+  /// named for the module, which already needs to be unique.
+  /// If the there is more than one instance of the node class then the
+  /// singleton tag should be Singleton post-pended with a unique identifier
+  /// for that specific node (e.g. the name).
```

Comment:
The current mrml nodes don't really follow the convention of the third case, the color nodes specifcally do not include the 'Singleton' string in the singleton tag, just the color node names generally, leading to IDs like:
name= MediumChartColors id = vtkMRMLColorTableNodeFileMediumChartColors.txt singleton tag =  FileMediumChartColors.txt
The updated docs I'm about to commit mention any new future nodes with an example of the convention to be followed in the future.